content/blog/2019-07-02-grainchanger.md

```diff
@@ -0,0 +1,183 @@
+---
+slug: grainchanger
+title: Aggregating spatial data with the grainchanger package
+package_version: 0.2.0
+authors:
+  - Laura Graham
+date: 2019-07-02
+categories: blog
+topicid:
+tags:
+- Software Peer Review
+- R
+- community
+- spatial
+- data-munging
+# delete the line below
+# if you have no preferred image
+# for Twitter cards
+twitterImg: img/blog-images/2019-07-02-grainchanger/logo.png
+output: 
+  html_document:
+    keep_md: true
+---
+
+
+
+The grainchanger package provides functionality for data aggregation to a coarser resolution via moving-window or direct methods.
+
+### Why do we need new methods for data aggregation?
+
+As landscape ecologists and macroecologists, we often need to aggregate data in order to harmonise datasets. In doing so, we often lose a lot of information about the spatial structure and environmental heterogeneity of data measured at finer resolution. 
+
+The issues around scale disconnects are both conceptual and practical: 
+
+#### Conceptual issues
+
+- The scale at which drivers affect responses varies
+
+- As does the resolution at which broad-scale patterns of ecological processes emerge
+
+#### Technical issues
+
+- The scale we measure this stuff at varies too
+
+- Ecological response data (e.g. species richness from species' atlas data) are often available at coarse resolutions, whereas with increasing availability of high-resolution remote sensing data, environmental data are available at much finer resolutions
+
+So how do we aggregate this fine-resolution data without losing information about spatial structure? We argue that using a moving window to calculate heterogeneity before aggregating retains information about spatial structure at an appropriate scale. We can then use this aggregated measure to (1) identify the appropriate scale-of-effect when we only have coarse-grain response data; and (2) investigate the effect of cross-scale interactions on ecological phenomena. See [our paper in Methods in Ecology & Evolution](https://doi.org/10.1111/2041-210X.13177)[^1] for simulations and a case study which show this. 
+
+We developed this method and R package in order to overcome some of these issues.
+
+### How does the method work? 
+
+The moving-window data aggregation (MWDA) method smooths an input raster using a specified function within a moving window of a specified size and shape prior to aggregation. 
+
+<!--html_preserve--> {{% figure src = "/img/blog-images/2019-07-02-grainchanger/schematic.jpg" width = "200" alt = "Schematic of the moving-window data aggregation approach" %}}
```

Comment:
The schematic looks too small in the [preview](https://deploy-preview-490--ropensci.netlify.com/blog/2019/07/02/grainchanger/). Please increase `width`

Reply:
I think this should be okay now. I wasn't able to see the preview.

Reply:
looks good now